### DISCUSSION TITLE
Error with using yolov5 model in Colab

### DISCUSSION THREAD

**Discussion by ashleys20** [ultralytics/yolov5#5743] [Q&A] [answered]
When I run this line of code in Google Colab to access the pretrained model for yolov5:

`model = torch.hub.load('ultralytics/yolov5', 'yolov5', force_reload=True)`

I receive this error:

`RuntimeError: Cannot find callable yolov5 in hubconf`
 
I have all of the necessary code to clone the repository and install the requirements.  I also have everything working fine for v3, so I am unsure what makes this different.  Any advice would be greatly appreciated.  Thank you!

**Comment by joangog** [comment, marked as answer]
I think in the second argument you need to define the specific yolov5 model you need, for example 'yolov5s', 'yolov5m', etc.

**Reply by glenn-jocher** [reply]
Correct
